src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,55 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { ChangeEvent } from 'react';
+import cn from 'classnames';
+
+import { centuries } from '../constants/centuries';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sexFronParams = searchParams.get('sex');
```

Comment:
Typo: The variable `sexFronParams` should be renamed to `sexFromParams` for consistency and clarity.